.github/workflows/unit-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: Test Coverage
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  coverage-report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
@mdial89f add an action to setup direnv, node, and node cache so instead of repeating those steps you can use: ` - uses: ./.github/actions/setup`